frontend/src/components/messageSection/messageCard/messageCard.tsx

```diff
@@ -119,8 +117,7 @@ export default class MessageCard extends BaseCard<Message, MessageCardProps, Mes
                 </div>
                 <div className="message-card-footer-container">
                     <div className="message-card-footer-text">
-                        {this.message.username}
-                        <Twemoji text={this.flag} />
+                        {this.message.username + ((this.country !== "")?" (" + this.country + ")":"")}
```

Comment:
Maybe instead of doing this at render time, maybe we can frontload at instantiation time? Not sure if it'll make a difference in load time, but probably be better for readability.

Reply:
already mentioned in the chat, we don't have username so go ahead and remove that.